src/v2/Components/SavedSearchAlert/CreateSavedSearchAlert.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react"
```

Comment:
If `SavedSearchAlert` isn't shared across other apps / components we should avoid putting it in the shared `v2/Components` library and instead co-locate it with the ArtworkFilter (or wherever most appropriate)

Reply:
As I know `SavedSearchAlert ` will be used by [collection](https://artsyproduct.atlassian.net/browse/FX-3551) page, [categories](https://artsyproduct.atlassian.net/browse/FX-3552) etc.
So I think we should leave it in Components

Reply:
👍